readme.txt

```diff
@@ -94,6 +94,8 @@ If you purchase an [Ultimate or Professional license](https://sugarcalendar.com/
 - **[Event URLs](https://sugarcalendar.com/downloads/event-urls/?utm_medium=referral&utm_source=website&utm_campaign=wporg-scl-plugin-page)** - Add a custom URL to any event created through Sugar Calendar with this add-on. This lets you redirect to events on other web sites and pages or simply open them in a new tab.
 - **[Advanced Recurring](https://sugarcalendar.com/preview-of-the-upcoming-advanced-recurring-add-on/?utm_medium=referral&utm_source=website&utm_campaign=wporg-scl-plugin-page)**  (coming soon!) - Create events that recur on specific days, every X period, recur X times, etc.
 
+Sugar Calendar is also a Stripe verified partner. The Stripe partner program recognizes applications which meet their strict quality requirements and allows us to provide even better services to our customers.
```

Comment:
> Sugar Calendar is also a Stripe verified partner.

I'd remove "also" here. Nothing before it is correspondingly similar.

> The Stripe partner program recognizes applications

When I first read this, I thought "applications" should have been "applicants", because it followed "verified partner" part.

> allows us to provide even better services to our customers

How _does_ being a verified partner allow us to provide even better services to our customers? I think this is worth elaborating on if it's worth mentioning at all.

----

What about...

> Sugar Calendar is a verified member of the [Stripe Partner Program](https://stripe.com/partner-program). This program recognizes organizations that meet Stripe's strict quality requirements, and provides users of Sugar Calendar and Stripe with the confidence they need to trust us both with their transactions.

Reply:
Good suggestions! Made a few more revisions in addition to yours.

Reply:
Had a write issue to this repo when I attempted a commit (see Basecamp ping). _Then_ these edits should show up.